StudCity.Frontend/src/app/features/authenticate.api.js

```diff
@@ -0,0 +1,31 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { AuthenticateService } from '../../services/authenticateService';
+import { showAlert } from '../../services/showAlert';
+import { googleAuthenticate } from '../../services/googleService';
+
+const authService = new AuthenticateService();
+
+export const googleLogin = createAsyncThunk(
+    'authenticate/googleLogin',
+    async (googleAccessToken, { fulfillWithValue, rejectWithValue }) => {
+        try {
+            const { name, picture, email, hd } = await googleAuthenticate(googleAccessToken);
+
+            const { isRegistration, isSuccess } = await authService.googleLogin({ name, picture, email, hd });
+			console.log(isSuccess, isRegistration);
```

Comment:
better to remove, but if it's for developing purpose then okay

Reply:
Agree with you! We can remove this line